build-system/tasks/changelog.js

```diff
@@ -437,11 +437,11 @@ function getClosedPullRequests(opt_page) {
   options.qs = {
     state: 'closed',
     page: opt_page,
-    access_token: GITHUB_ACCESS_TOKEN,
```

Comment:
Could we change the policy to align with our non-build code?

Reply:
We currently use the rules in `eslint-config-es5` because the build system code is written in ES5 JS, while the runtime is in ES 6. That's where the non-conforming comma rules are coming from.

I'm working on a parallel thread on upgrading to using node 6 on Travis, at which point we can safely use ES6 code in `build-system`, and then use the same linter as the rest of the runtime. Will update this PR once I have that part figured out.

Reply:
We now use the same ES6 rules as the rest of the AMP runtime, and trailing commas are back. PTAL.